python/lsst/ap/verify/ap_verify.py

```diff
@@ -137,7 +139,17 @@ def _measure_final_properties(metadata, metrics_job):
     metrics_job: `verify.Job`
         The Job object to which to add any metric measurements made.
     """
-    measurements = measure_from_metadata(metadata)
+    measurements = []
+    measurements.extend(measure_from_metadata(metadata))
+    # In the current version of ap_pipe, DIFFIM_DIR has a parent of
+    # PROCESSED_DIR. This means that a butler created from the DIFFIM_DIR reop
+    # includes data from PROCESSED_DIR.
+    print(args)
```

Comment:
The print statement looks like it's left over from debugging. Should this function always be verbose?